From d8ed757c2856415a20e13c405f902fff261fcea8 Mon Sep 17 00:00:00 2001
From: reichi <reichi@opendreambox.org>
Date: Fri, 8 Nov 2019 10:07:28 +0100
Subject: [PATCH] SystemPlugins/HdmiCec: reset self._skip_next_poweroff_message
 after we've been reactivated - fixes poweroff working only every second time
 with a bunch of TVs

Change-Id: Ib87d53ec031c632d3be657929e84bdb5dcdd519d
---
 usr/lib/enigma2/python/Plugins/SystemPlugins/HdmiCec/plugin.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/usr/lib/enigma2/python/Plugins/SystemPlugins/HdmiCec/plugin.py b/usr/lib/enigma2/python/Plugins/SystemPlugins/HdmiCec/plugin.py
index 8dcda3f..c0b85ac 100644
--- a/usr/lib/enigma2/python/Plugins/SystemPlugins/HdmiCec/plugin.py
+++ b/usr/lib/enigma2/python/Plugins/SystemPlugins/HdmiCec/plugin.py
@@ -55,6 +55,7 @@ class Cec(object):
 				self.session.open(Standby)
 
 	def __receivedNowActive(self):
+		self._skip_next_poweroff_message = False
 		if config.cec.receivepower.value:
 			from Screens.Standby import inStandby
 			if inStandby != None:
@@ -63,6 +64,7 @@ class Cec(object):
 			self._skip_next_poweron_message = True
 
 	def powerOn(self, forceOtp=False):
+		self._skip_next_poweroff_message = False
 		if self._skip_next_poweron_message:
 			self._skip_next_poweron_message = False
 			return
-- 
2.1.4

